docs/content/any/getting-started/quickstart/index.md

```diff
@@ -1,192 +1,246 @@
-  If a user's email doesn't end in `"@example.com"`, the rule fails, and they
-  are denied access:
+app = Flask(__name__)
+@app.route("/some/page/<pagenum>")
+def page_show(pagenum):
+   page = Page.pages[int(pagenum)]
+   if oso.is_allowed(
+       get_user(), # the user doing the request
+       "read", # the action we want to do
+       page): # the resource we want to do it to
 
-```console
-$ curl -H "user: alice@foo.com" localhost:5050/expenses/1
-Not Authorized!
+       return f'<h1>A Page</h1><p>this is page {pagenum}</p>'
+   else:
+       return f'<h1>Sorry</h1><p>You are not allowed to see this page</p>'
 ```
 
-  If you aren’t seeing the same thing, make sure you created your policy
-  correctly in `expenses.polar`.
-{{% /callout %}}
-
-## Using application data
-
-We now have some basic access control in place, but we can do better.
-Currently, anyone with an email ending in `@example.com` can see all expenses —
-including expenses submitted by others.
-
-{{% callout "Edit it!" "blue" %}}
-  Let's modify our existing rule such that users can only see their own
-  expenses:
-
-  {{< literalInclude dynPath="expensesPath2"
-                     fallback="expenses2" >}}
-{{% /callout %}}
+Oso denies requests unless you explicitly tell it to accept that sort of request.
+You can tell Oso what requests to accept by providing it with a file full of rules, which we call a ‘policy’.
 
-Behind the scenes, Oso looks up the `submitted_by` field on the provided
-`Expense` instance and compares that value against the provided **actor**. And
-just like that, an actor can only see an expense if they submitted it!
+## Write an authorization policy
+Here is a typical policy, written in our declarative language, **Polar**.
+It lets any actor with the role `user` read a page, but only lets actors with the role `admin` write to a page.
 
-{{% callout "Try it!" "green" %}}
-  Alice can see her own expenses but not Bhavik's:
+We can load our example policy from a file with the extension `.polar`.
 
-```console
-$ curl -H "user: alice@example.com" localhost:5050/expenses/1
-Expense(...)
+```python
+oso.load_file("example.polar")
 ```
 
-```console
-$ curl -H "user: alice@example.com" localhost:5050/expenses/3
-Not Authorized!
+Here’s the example.polar file:
+
+```polar
+actor_role(actor, role) if
+   resources = Page.pages and
+   r in resources and
+   actions = r.has_roles(actor) and
+   action in actions and
+   role = { name: action, resource: r };
+
+resource(_type: Page, "page", actions, roles) if
+   actions = ["read", "write"] and
+   roles = {
+       admin: {
+           permissions: ["write"],
+           implies: ["user"]
+       },
+       user: {
+           permissions: ["read"]
+       }
+   };
+ ```
+
+An _actor_role_ rule expresses the relationship between an actor and a role object of the form `{name: “the-role-name”, resource: TheResourceObject}`.
+The `is_allowed` call in your Python code will look up this rule, so this is required.
```

Comment:
```suggestion
This is required so that Oso can lookup the roles assigned to an actor to decide whether to accept a request.
```

Adding this suggestion because `is_allowed` does not require this. It's required because we called `enable_roles()`.